hypha/apply/projects/models/project.py

```diff
@@ -372,13 +361,7 @@ def start_date(self):
 
     @property
     def end_date(self):
-        # Aiming for the proposed end date as the last day of the project
-        # If still ongoing assume today is the end
-        if self.proposed_end:
-            return max(
-                self.proposed_end.date(),
-                timezone.now().date(),
-            )
+        # Assume today is the end
```

Comment:
I think it would be great to include a little explanation on why `today` is chosen as the end date.

Reply:
Good point.